cmd/k8s-deployment-scaler/main.go

```diff
@@ -245,8 +241,7 @@ func handleGetReplicaCount(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Get the deployment from the cache
-	info, exists := getDeploymentFromCache(namespace, deploymentName)
+	deployment, exists := getDeploymentFromCache(namespace, deploymentName)
 	if !exists {
 		writeJSONError(w, apiError{
 			Message: "Deployment not found in cache",
```

Comment:
```
➜  k8s-deployment-scaler git:(development/helm) curl -X GET "https://localhost:8443/replica-count?namespace=default&deployment=lallala" --cert ./certs/client-cert.pem --key ./certs/client-key.pem --cacert ./certs/ca-cert.pem
{"message":"Deployment not found in cache","code":404}
```

This is the error I get when trying to query deployment that doesn't exist. 
Clients doesn't really need to know about the cache, just if deployment exist or not.

Reply:
Agreed, I will update this.